docs/content/en/docs/getting-started/ports/_index.md

```diff
@@ -39,7 +66,7 @@ Use the following to access the SSH service on the control plane and etcd nodes:
 | TCP      | Inbound   | 22         | SSHD server             | SSH clients               |
 
 
-## Kubernetes worker nodes
+##### Kubernetes worker nodes
```

Comment:
```suggestion
### Kubernetes worker nodes
```
I suggested changing the heading above to ##. If you agree, then this should be ###.